src/dom.js

```diff
@@ -509,3 +509,13 @@ export function isJsonScriptTag(element) {
   return element.tagName == 'SCRIPT' &&
             element.getAttribute('type').toUpperCase() == 'APPLICATION/JSON';
 }
+
+
+export function insertAfter(newNode, referenceNode) {
+  const nextSibling = referenceNode.nextSibling;
+  if (nextSibling) {
```

Comment:
No need for the check, using `insertBefore(newNode, null)` will append, and `nextSibling = null` if there's not a sibling.